client/app/bundles/course/assessment/skills/pages/SkillsIndex/index.tsx

```diff
@@ -148,7 +148,7 @@ const SkillsIndex: FC<Props> = (props) => {
   return (
     <>
       <PageHeader title={intl.formatMessage({ ...translations.skills })} />
-      <Grid container direction="row" columnGap={0.2} maxHeight="70vh">
+      <Grid container direction="row" columnGap={0.2}>
```

Comment:
If there are a lot of skills/skill branches, do you need to scroll below to see the description popup?

Reply:
Ah ok thats fine since the maxheight is controlled by the table instead.